apps/examples/wifi_manager/wm_test/Kconfig

```diff
@@ -8,6 +8,7 @@ config EXAMPLES_WIFIMANAGER_TEST
 	default n
 	depends on WIFI_MANAGER
 	depends on NET_LWIP_DHCP
+	depends on NET_SECURITY_TLS
```

Comment:
If this dependency is added TLS which is heavy will be mandatory feature of wm_test. could you make optional easy setup stress test in wm_test?

Reply:
Changes Done as suggested.